contracts/contracts/EventManager.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+contract EventManager {
```

Comment:
i'm wondering if you could mark the contract as `abstract`, so it could be only used as a "base" for other contracts and cannot be instantiated https://docs.soliditylang.org/en/v0.6.2/contracts.html#abstract-contracts